test/unit/measure_validator_test.rb

```diff
@@ -53,12 +69,6 @@ class MeasureValidatorTest < ActiveSupport::TestCase
     assert_equal 2, errors.length
   end
 
-  test 'should have no errors if set id is valid (cat 1)' do
```

Comment:
What was the reasoning behind removing this test? (Not saying it shouldn't be removed, just curious to know why)

Reply:
It was unclear to me what differentiated this test from the 'should have no errors if measure id is valid (cat 1)' test.

Reply:
It was unclear to me what differentiated this test from the 'should have no errors if measure id is valid (cat 1)' test.